CITATION.cff

```diff
@@ -0,0 +1,21 @@
+cff-version: 1.2.0
+message: "If you use Mantid in your research, please cite it using this metadata."
+abstract: "The Mantid project provides a framework that supports high-performance computing and visualisation of scientific data. Mantid has been created to manipulate and analyse Neutron and Muon scattering data, but could be applied to many other techniques. The framework is open source and supported on multiple target platforms (Windows, Linux, Mac OS X)."
+authors:
+  - name: "The Mantid Project team"
+    email: mantid-help@mantidproject.org
+    website: https://www.mantidproject.org/
+title: Manipulation and Analysis Toolkit for Instrument Data
+version: 6.2.0
```

Comment:
Currently there is not a plan, but its a good question. I don't think there is a mechanism to update the info in these files automatically (at the moment at least). 

I can see that having the exact nightly version of a software would be useful in terms of ensuring reproducibility of a particular workflow, however citing a potentially unstable version of Mantid could be seen as a bad thing at the same time. Its probably worth getting the opinion of a gatekeeper on this. For now, I think just having an up-to-date major, minor and patch number would be good enough. Which reminds me, I might need to update the Patch Release checklist - I will check

Reply:
I suppose part of the issue would be with where we present that information too. If the citation is appearing on the repo should it represent the current state of the repo or the last full release? Does this just appear on the repo or is it accessible from within mantid too?

Reply:
I would say that it should represent the last release (including patches), but again its worth a gatekeeper weighing in on this.

For now it just appears on the repo. However it will be possible to access this data from within Mantid if we want to by using https://pypi.org/project/cffconvert/ . One concern I had about accessing this info from within Mantid is that the DOI is generated after the release candidate has been finalised (i.e. all changes to the software are now in). Meaning the DOI in the CITATIONS file that is bundled into the release and accessed from within Mantid would be outdated.

Reply:
Oh yes, for sure, I think I was mostly just thinking aloud (or into a text field, as it were). 

I see, that makes sense. Perhaps having a link to the citations file on the website next to the current and previous releases would make sense in that case. But like you said, probably something for a gatekeeper to weigh in on.